requirements.txt

```diff
@@ -1,47 +1,46 @@
 # Autogenerated by 'make bs'
 # If the format has changed, update MIN_PIPENV_VERSION in Makefile
 -i https://pypi.org/simple
-bracex==2.3.post1; python_version >= '3.7'
-cairocffi==1.5.1; python_version >= '3.7'
-cairosvg==2.7.0
-certifi==2023.5.7; python_version >= '3.6'
-cffi==1.15.1
-charset-normalizer==3.1.0; python_full_version >= '3.7.0'
-click==8.1.3; python_version >= '3.7'
+bracex==2.4; python_version >= '3.8'
+cairocffi==1.6.1; python_version >= '3.7'
+cairosvg==2.7.0; python_version >= '3.5'
+certifi==2023.7.22; python_version >= '3.6'
+cffi==1.16.0; python_version >= '3.8'
+charset-normalizer==3.3.0; python_full_version >= '3.7.0'
+click==8.1.7; python_version >= '3.7'
 colorama==0.4.6; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5, 3.6'
 cssselect2==0.7.0; python_version >= '3.7'
 defusedxml==0.7.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
 ghp-import==2.1.0
 idna==3.4; python_version >= '3.5'
-importlib-metadata==6.6.0
+importlib-metadata==6.6.0; python_version >= '3.7'
 jinja2==3.1.2; python_version >= '3.7'
-lxml==4.9.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
 markdown==3.3.7; python_version >= '3.6'
-markupsafe==2.1.2; python_version >= '3.7'
+markupsafe==2.1.3; python_version >= '3.7'
 mergedeep==1.3.4; python_version >= '3.6'
-mkdocs==1.4.3
-mkdocs-awesome-pages-plugin==2.9.1
-mkdocs-macros-plugin==1.0.1
-mkdocs-material==9.1.15
+mkdocs==1.4.3; python_version >= '3.7'
+mkdocs-awesome-pages-plugin==2.9.1; python_version >= '3.7'
+mkdocs-macros-plugin==1.0.1; python_version >= '3.8'
+mkdocs-material==9.1.15; python_version >= '3.7'
 mkdocs-material-extensions==1.1.1; python_version >= '3.7'
-mkdocs-redirects==1.2.0
+mkdocs-redirects==1.2.0; python_version >= '3.6'
 mkdocs-simple-hooks==0.1.5
-natsort==8.3.1; python_version >= '3.7'
-packaging==23.1; python_version >= '3.7'
-pillow==9.5.0
+natsort==8.4.0; python_version >= '3.7'
+packaging==23.2; python_version >= '3.7'
+pillow==9.5.0; python_version >= '3.7'
 pycparser==2.21
-pygments==2.15.1; python_version >= '3.7'
-pymdown-extensions==10.0.1; python_version >= '3.7'
+pygments==2.16.1; python_version >= '3.7'
+pymdown-extensions==10.3; python_version >= '3.8'
 python-dateutil==2.8.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-pyyaml==6.0; python_version >= '3.6'
+pyyaml==6.0.1; python_version >= '3.6'
 pyyaml-env-tag==0.1; python_version >= '3.6'
-regex==2023.5.5; python_version >= '3.6'
+regex==2023.10.3; python_version >= '3.7'
```

Comment:
Note: I haven't pinned all the indirect dependencies, which doesn't seem worth the trouble for now but we can always do it in the future if/when we want to